src/ripple/protocol/impl/TxFormats.cpp

```diff
@@ -113,6 +113,26 @@ TxFormats::TxFormats ()
         << SOElement (sfSignerQuorum,        SOE_REQUIRED)
         << SOElement (sfSignerEntries,       SOE_OPTIONAL)
         ;
+
+    add ("PaymentChannelCreate", ttPAYCHAN_CREATE) <<
+            SOElement (sfDestination,       SOE_REQUIRED) <<
+            SOElement (sfAmount,            SOE_REQUIRED) <<
+            SOElement (sfSettleDelay,       SOE_REQUIRED) <<
+            SOElement (sfPublicKey,         SOE_REQUIRED) <<
+            SOElement (sfCancelAfter,       SOE_OPTIONAL) <<
+            SOElement (sfDestinationTag,    SOE_OPTIONAL);
```

Comment:
I'd suggest allowing `sfInvoiceID` as an optional field as well: it's a `uint256` which is _almost_ orthogonal to the destination tag and affords a lot more space for tracking multiple concurrent flows and associating a particular channel with a unique, user-selectable identifier.